libs/device-core/src/commands/use-cases/app-backup/getAppStorageInfo.ts

```diff
@@ -0,0 +1,82 @@
+import Transport, { StatusCodes, TransportStatusError } from "@ledgerhq/hw-transport";
+import { LocalTracer } from "@ledgerhq/logs";
+import type { AppStorageInfo } from "../../entities/AppStorageInfo";
+import type { APDU } from "../../entities/APDU";
+
+/**
+ * Name in documentation: INS_APP_STORAGE_GET_INFO
+ * cla: 0xe0
+ * ins: 0x6a
+ * p1: 0x00
+ * p2: 0x00
+ * data: APP_NAME_LEN (1 byte) + APP_NAME (variable)
+ */
+const GET_APP_STORAGE_INFO: APDU = [0xe0, 0x6a, 0x00, 0x00, undefined];
+
+/**
+ * 0x9000: Success.
+ * 0x5123: Application not found.
+ * 0x662F: If the device is in recovery mode.
+ * 0x670A: Invalid application name length, two chars minimum.
+ */
+const RESPONSE_STATUS_SET: number[] = [
+  StatusCodes.OK,
+  StatusCodes.APP_NOT_FOUND_OR_INVALID_CONTEXT,
+  StatusCodes.CUSTOM_IMAGE_BOOTLOADER,
```

Comment:
[SHOULD] It's not a good idea to reuse `CUSTOM_IMAGE_BOOTLOADER`, rather add a new entry with the same value.
It doesn't have the same meaning: in the context of Custom Lock Screen it's when the device is in bootloader mode while here it's when the device is in recovery mode.

Reply:
Adding new entry with the same value has nothing to do to clarify the meaning. The error will always [find the first match value](https://github.com/LedgerHQ/ledger-live/blob/5d17fab23f59b888190cd9e122165be6d0eb8c55/libs/ledgerjs/packages/errors/src/index.ts#L320) then return its key. So either differentiating the value at embedded team or using a different error system (with context) can be the solution.

Reply:
ok 👍 indeed our system isn't perfect